frontend/packages/dev-console/integration-tests/features/vulnerability/project_vulnerability_ODC-4882

```diff
@@ -0,0 +1,104 @@
+Feature: Image Vulnerability for each Project
+    As a user I would like to see Image Vulnerability for each Project
+
+Background:
+    Given user is in Developer perspecitve
+    And user in project namespace
```

Comment:
Didn't get this statement. You didn't mention the project name.

Reply:
Updated